src/pages/workspaces/workspace/analysis/modals/ComputeModal.js

```diff
@@ -22,14 +22,15 @@ import * as Nav from 'src/libs/nav'
 import { useOnMount } from 'src/libs/react-utils'
 import * as Style from 'src/libs/style'
 import * as Utils from 'src/libs/utils'
+import { getPersistentDiskCostMonthly, runtimeConfigBaseCost, runtimeConfigCost } from 'src/pages/workspaces/workspace/analysis/cost-utils'
 import { WarningTitle } from 'src/pages/workspaces/workspace/analysis/modals/WarningTitle'
-import { SaveFilesHelp, SaveFilesHelpRStudio } from 'src/pages/workspaces/workspace/analysis/runtime-common'
+import { computeStyles, RadioBlock, SaveFilesHelp, SaveFilesHelpRStudio } from 'src/pages/workspaces/workspace/analysis/runtime-common'
 import {
-  computeStyles, defaultAutopauseThreshold, defaultComputeRegion, defaultComputeZone, defaultDataprocMachineType, defaultDataprocMasterDiskSize,
+  defaultAutopauseThreshold, defaultComputeRegion, defaultComputeZone, defaultDataprocMachineType, defaultDataprocMasterDiskSize,
   defaultDataprocWorkerDiskSize, defaultGceBootDiskSize, defaultGcePersistentDiskSize, defaultGpuType, defaultLocation,
   defaultNumDataprocPreemptibleWorkers, defaultNumDataprocWorkers, defaultNumGpus, defaultPersistentDiskType, displayNameForGpuType, findMachineType, getAutopauseThreshold,
-  getDefaultMachineType, getIsRuntimeBusy, getPersistentDiskCostMonthly, getValidGpuOptions, getValidGpuTypesForZone,
-  isAutopauseEnabled, pdTypes, RadioBlock, runtimeConfigBaseCost, runtimeConfigCost
+  getDefaultMachineType, getIsRuntimeBusy, getValidGpuOptions, getValidGpuTypesForZone,
+  isAutopauseEnabled, pdTypes
 } from 'src/pages/workspaces/workspace/analysis/runtime-utils'
```

Comment:
@mlilynolting in my work (https://github.com/DataBiosphere/terra-ui/pull/3765), I am seeing that [line 744](https://github.com/DataBiosphere/terra-ui/pull/3724/files#diff-a15b61bc748955fee99cca754569a25705e76701992dfc9da20bafbd7dffb664R744) in this file is a place where you need to handle that `location` will initially be undefined. Thankfully this was caught by an integration test failure.

https://circleci-tasks-prod.s3.us-east-1.amazonaws.com/storage/artifacts/1c1bde7e-0f01-4eef-b2bc-01f4b12ed651/43a6c556-5405-4d6f-bfd2-98dda869136f/1/results/failure-screenshots/failureDetails-1675434751249-run-rstudio.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAQVFQINEOMS5D5RH4%2F20230203%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20230203T143651Z&X-Amz-Expires=60&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEHcaCXVzLWVhc3QtMSJIMEYCIQCmY7Dsw2LSZFuZbnlPX0WPgg9eHbA2NW7AvqkiWTpccAIhAPsyOQM%2FnwD7eu1uaVX6V0BOyHCz3nK9TbeczjF1etE1KrQCCPD%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEQAxoMMDQ1NDY2ODA2NTU2Igy6ExGjKRq214Y8MPYqiALsTfkdOAfQ8FXASfrWcDjBigNsOtJ8vIqDUkFSxiV4%2B5FE3iZVT3egS2p1u4bYGm6jmlRRkvM0ZcchHPJSzE2YFUmtkm%2BmM1WqZGYyBoTUkEwqf%2Ft8lRfqKahXHjaH6VgcVITDvGgDx6PKeBaKrH9ntPxQT75kRQQ3W5uhsYWfscwN1y6K18bulyhF6auz4ms%2BS%2BB5jIzmn14qUPXh9d7jrr6nYlBQ36SOXv%2FWaxvDys2DnpXLCL3DF3FsfYhJgJN%2FeHRuZaRpuq171vezPNK%2FNrcREXdfV8ZbPXbPdxZ%2F5oKgyYYIDUX18RkwS3Xfj%2BBlAFrG%2FfCPbcKFR5o8apbvsVE%2FrWH9sHMw3rf0ngY6nAGPtiHk1GuD5vULA%2BBSDUELyTUcYUIEDNo010us2u1zxkz%2BTxcb9Hub5r%2Fmw7nmuYmK7f3eoMrxLrx4FrRzwMwIhWmiozPZOapFvgvmc%2BLbrQFU6mM%2BeiJ47j026dwChwdiPeIwHQ19yVNzycJcMBbhnT6yp8z%2FG7AYHAp%2FCI4%2FkktdxT6DlbKX%2BaHJAgqqILNku6dot7kavCLJumw%3D&X-Amz-SignedHeaders=host&x-id=GetObject&X-Amz-Signature=811f4f13d643952d123cf5caf3d764068fc01dc9cb9acfdc35d658f48b3270a0